packages/ui/Drawing/Drawing.js

```diff
@@ -393,6 +415,26 @@ const DrawingBase = kind({
 			}
 		}, [drawingTool]);
 
+		// Handle width and height of canvas dynamically
+		useEffect(() => {
+			const canvas = canvasRef.current;
+			const context = canvas.getContext('2d');
+			const handleResize = () => {
+				canvas.height = ri.scale(canvasHeight);
+				canvas.width = ri.scale(canvasWidth);
+				canvas.style.height = `${canvasHeight}px`;
+				canvas.style.width = `${canvasWidth}px`;
```

Comment:
You should add ri.scale here also. Otherwise the drawing is broken in Sandstone
```suggestion
				canvas.style.height = `${ri.scale(canvasHeight)}px`;
				canvas.style.width = `${ri.scale(canvasWidth)}px`;
```